packaging/docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "2"
+
+services:
+  ultimate-webbackend:
+    build:
+      dockerfile: "Dockerfile"
+      target: "ultimate-webbackend"
+    image: "ultimate-webbackend"
+    env_file: "ultimate-webbackend.env"
+    ports:
+      - "8080:8080"
```

Comment:
The port of the backend should not be hard-coded here.
Instead, the value from the Docker Compose environment file should be used.